accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -495,10 +498,24 @@ of a function call produces a result of type `Never`.
 
 The static type of a `throw e` expression is `Never`.
 
-In legacy mode, an override of operator== with no explicit parameter type
+Consider an expression `e` of the form `e1 == e2` where the static type of
+`e1` is `T1` and the static type of `e2` is `T2`. Let `S` be the type of the
+formal parameter of `operator ==` in the interface of **NonNull**(`T1`).
```

Comment:
This contradicts the text beginning on line 485.  I'm ok with making this change (to treat == specially), but I really don't want to leave this kind of trap for readers of the spec.  The text around line 485 should call out that there is an exception specified below (e.g. "With the exception of operator== which is specified separately below, the type used for static analysis.... ").

Reply:
I added a commentary paragraph just after line 485 to make it explicit that `e1 == e2` is not just an invocation of the instance member `operator ==`, and that the precise rules are specified later in the same section.